Migrate to symfony messenger (#2)
Setono/SyliusCalloutsPlugin: README.md

* `bottom_right_corner`
* `bottom_left_corner`

7. Configure Async assigns product callouts

>     This plug assigns product callouts when the administrator create/update the product or callout, with a large number of products this can result in slower page performance. To circumvent that you can use RabbitMQ with Symfony Messenger to assigns product callouts.

Reviewer: ```suggestion
    This plugin assigns product callouts when the administrator create/update the product or callout. With a large number of products this can result in slower page performance. To circumvent this problem you can use an async transport with Symfony Messenger to assigns product callouts.
```

    
    Follow the installation instructions here: [How to Use the Messenger](https://symfony.com/doc/current/messenger.html) and then [configure a transport](https://symfony.com/doc/current/messenger.html#transports).
    
    Basically you should do:
    ```bash
    $ composer req symfony/messenger symfony/serializer-pack
    ```
    
    Then configure the Messenger component:
    ```yaml
    # config/packages/messenger.yaml
    framework:
        messenger:
            transports:
                amqp: "%env(MESSENGER_TRANSPORT_DSN)%"
    ```
    
    ```yaml
    # .env
    ###> symfony/messenger ###
    MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
    ###< symfony/messenger ###
    ```
    
    And finally configure the plugin to use your transport:
    
    ```yaml
    setono_sylius_callouts:
        messenger:
            transport: amqp
    ```
    

>     After this, the Messenger will be automatically enabled in this plugin and subsequently, it will assign asynchronous product callouts.

Reviewer: ```suggestion
    After this, the Messenger will be automatically enabled in this plugin and subsequently, it will assign product callouts asynchronously.
```
